docs/semgrep-sc/sc-getting-started.md

```diff
@@ -167,4 +167,28 @@ Semgrep Supply Chain enables developers to perform triage and remediation throug
 
 For more information, see [Triaging and remediating findings](/docs/semgrep-sc/triaging-and-remediating-vulnerabilities).
 
+## Appendix: Setting up SSC scans for specific project management tools
+
+### Apache Maven (Java)
+
+Semgrep Supply Chain does not read `pom.xml` files to parse Maven projects. Instead it parses a dependency tree generated by Maven (`mvn`). Perform the following steps to enable Semgrep Supply Chain to correctly parse Maven projects:
+
+<ol>
+    <li>Generate a file outlining the project's dependency tree by adding the following command to your build pipeline:<br />
+    <pre class="language-bash">mvn dependency:tree -DoutputFile=maven_dep_tree.txt</pre>
+    For specific steps to add the command into your build pipeline, refer to your CI provider's documentation.</li>
+    <li>For monorepositories or similar directory structures, create additional dependency trees per project subdirectory. Semgrep Supply Chain can detect and parse them all.</li>
+    <li>Run the Semgrep workflow, action, or step after the dependency tree or trees have been generated.</li>
+</ol>
```

Comment:
I think we should more clearly state that you need to generate one of these for every single `pom.xml` whose dependencies you want understood by semgrep. My impression is that even non-monorepo projects sometimes have multiple `pom.xml` files. Basically instead of "add this one command, oh and also you have to do it multiple times for monorepos" I'd say "for each pom.xml file in your project, generate a dependency tree for it using this command"

Reply:
Understood! Changed those bits.